app/main.py

```diff
@@ -10,6 +10,7 @@ def print_name(self):
     def feed(self):
         print(f"Eating {self.appetite} food points...")
         if self.is_hungry is True:
```

Comment:
Maybe you should check to see if the animal is hungry first and then feed it

Reply:
Good point!